librecad/src/main/main.cpp

```diff
@@ -294,6 +296,8 @@ int main(int argc, char** argv)
     setlocale(LC_NUMERIC, "C");
 
     RS_DEBUG->print("main: loading files..");
+    // get the file list from LC_Application
+    fileList << app.fileList();
```

Comment:
This is the only place where `fileList()` is called.
At this time `LC_Application` is instanced and initialized, but the event loop is not running and so `LC_Application::files` is still empty.